src/utils/hact/workOrderComplete/closeWorkOrder.js

```diff
@@ -26,6 +27,36 @@ export const buildCloseWorkOrderData = (
       },
     ],
   }
+
+  if (followOnRequest !== null) {
+    dataObject['followOnRequest'] = followOnRequest
+  }
+
+  return dataObject
+}
+
+export const buildFollowOnRequestData = (
```

Comment:
Is this just to help with enforcing the schema?

Reply:
Yeah, makes it more explicit given there is no typescript